pkg/apis/core/v1/defaults.go

```diff
@@ -438,3 +446,56 @@ func SetDefaults_PodLogOptions(obj *v1.PodLogOptions) {
 		}
 	}
 }
+
+// defaultPodRequests applies default values for pod-level requests, only when
+// pod-level limits are set, in following scenarios:
+// 1. When at least one container (regular, init or sidecar) has requests set:
+// The pod-level requests become equal to the effective requests of all containers
```

Comment:
nit: I think it will be good to copy relevant rows from the table we have in the KEP as here.

Reply:
Ack'ing for next release